mozdownload/__init__.py

```diff
@@ -2,4 +2,15 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from scraper import *
+from scraper import PLATFORM_FRAGMENTS, NotSupportedError, NotFoundError, \
+    NotImplementedError, TimeoutError, Scraper, DailyScraper, \
+    DirectScraper, ReleaseScraper, ReleaseCandidateScraper, \
+    TinderboxScraper, TryScraper
+from parser import DirectoryParser
+from timezones import PacificTimezone
```

Comment:
As a normal user you don't need those. If necessary they need to be imported when needed.